netbox_slm/models.py

```diff
@@ -113,12 +114,25 @@ class SoftwareLicense(NetBoxModel):
         null=True,
         blank=True
     )
+    license_key = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
Requirements of #8 stated that the key should not actually be stored in netbox (it would typically be vaulted somewhere).

Reply:
Haven't seen #8 Will be removed